### DISCUSSION TITLE
How used inline css inside widget

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#10667] [Q&A]
![Imageshowing](https://user-images.githubusercontent.com/5202291/140777382-ce44f4ba-2032-4096-8f69-53f5eb453c56.png)

**Comment by jptissot** [comment]
Maybe you should use the HtmlBody part for this. This looks like Html and not markdown.

**Reply by saleemhaq** [reply]
So I create Widget based on HtmlBody part?

**Reply by ns8482e** [reply]
@saleemhaq  `Markdown != Html`  you are using Markdown body part but if you need to use `<html>` tags use `HtmlBodyPart` instead

**Reply by saleemhaq** [reply]
In HtmlBody from admin side how i put dynamic images it not allowed the media part?

**Reply by saleemhaq** [reply]
> `HtmlBodyPart`

Now i used the HtmlBodyPart in line below:

<section class="inner-banner-wrap" style="background-image: url(../images/software-banner.jpg);">

but it show the plain line of inline CSS how i achieve with dynamic images with that?

![html-Body](https://user-images.githubusercontent.com/5202291/140797946-45d53ec6-4168-4162-9541-1b12b56f19fa.PNG)

**Reply by ns8482e** [reply]
`[image]` is to generate `<img>` tag - I guess you need `asset_url`  - Search documentation for `Shortcodes` and enable related feature if you have not already
